pkg/reconciler/testing/configmap.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2021 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testing
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	"github.com/ghodss/yaml"
```

Comment:
Can we use sigs.k8s.io/yaml instead? It is a fork of ghodss/yaml so should have the same behavior

See https://github.com/tektoncd/triggers/pull/1110#discussion_r681232522

Reply:
Updated :+1: